### ISSUE TITLE
output 관련 질문

### ISSUE THREAD

**Issue by YSIT21013** [2021-SIT-AdvancedAlgorithms/PA4_Graph#3]
주어진 output_1.txt에서는 output이 islands[island_idx][idx]의 idx로 나오도록 되어있는데, islands[island_idx][idx].index로 나와도 되나요?

**Comment by sungminlee114** [comment]
vertex 고유 인덱스로 출력해도 되냐는 뜻이죠?

**Comment by YSIT21013** [comment]
네 예를 들어 islands[1]의 경우 output이 0, 1, 2, 0, 3, 4, 0으로 되어있는데, 대신 5, 6, 7, 5, 8, 9, 5를 출력해도 되는 것인가요

**Comment by sungminlee114** [comment]
아뇨 통일시켜주시면 감사하겠습니다. 디버깅시 헷갈리나요?

**Comment by YSIT21013** [comment]
아니요. 다만 output을 제대로 확인하지 않고 처음에 코딩하였을 때의 출력이 바로 5, 6, 7, 5, 8, 9, 5를 출력하는 것이었습니다. 현재는 주어진 output과 동일하게 출력하도록 수정하였습니다.
